modules/Core/language/en_UK.json

```diff
@@ -1366,5 +1365,10 @@
     "user/banned_from_registering": "Your IP has been banned, you cannot register.",
     "user/you_have_received_a_warning": "You have received a warning!",
     "user/group_has_been_added": "Group {{group}} has been added to user {{user}}",
-    "user/group_has_been_removed": "Group {{group}} has been removed from user {{user}}"
+    "user/group_has_been_removed": "Group {{group}} has been removed from user {{user}}",
+    "user/validate_email": "Validate Email",
+    "user/change_or_resend_email": "Change or resend email",
+    "user/validate_email_info": "Thanks for registering! You have been sent an email to <strong>{{email}}</strong> with a link to activate your account. If you are unable to find the email, check your junk folder.",
```

Comment:
The second line would read better as "An email has been sent to <strong>{{email}}</strong>...` imo